panels/user-accounts/cc-user-panel.c

```diff
@@ -795,6 +808,39 @@ get_autologin_possible (ActUser *user)
 static void on_permission_changed (CcUserPanel *self);
 static void full_name_edit_button_toggled (CcUserPanel *self);
 
+#ifdef HAVE_MALCONTENT
+static gboolean
+is_parental_controls_enabled_for_user (ActUser *user)
+{
+        g_autoptr(MctManager) manager = NULL;
+        g_autoptr(MctAppFilter) app_filter = NULL;
+        g_autoptr(GDBusConnection) system_bus = NULL;
+        g_autoptr(GError) error = NULL;
+
+        /* FIXME: should become asynchronous */
+        system_bus = g_bus_get_sync (G_BUS_TYPE_SYSTEM, NULL, &error);
+        if (system_bus == NULL) {
+	        g_warning ("Error getting system bus while trying to show user details: %s", error->message);
+	        return FALSE;
+        }
+
+        manager = mct_manager_new (system_bus);
+        app_filter = mct_manager_get_app_filter (manager,
+                                                 act_user_get_uid (user),
+                                                 MCT_GET_APP_FILTER_FLAGS_NONE,
+                                                 NULL,
+                                                 &error);
+        if (error) {
```

Comment:
Probably want to ignore (don’t print a warning for) `MCT_MANAGER_ERROR_DISABLED` which is emitted when parental controls are globally disabled (e.g. due to not being installed properly).

Reply:
Addressed (diff below for easier review)

```diff
diff --git a/panels/user-accounts/cc-user-panel.c b/panels/user-accounts/cc-user-panel.c
index b2c4303d5..fb7d242e5 100644
--- a/panels/user-accounts/cc-user-panel.c
+++ b/panels/user-accounts/cc-user-panel.c
@@ -831,9 +831,11 @@ is_parental_controls_enabled_for_user (ActUser *user)
                                                  NULL,
                                                  &error);
         if (error) {
-                g_warning ("Error retrieving app filter for user %s: %s",
-                           act_user_get_user_name (user),
-                           error->message);
+                if (!g_error_matches (error, MCT_MANAGER_ERROR, MCT_MANAGER_ERROR_DISABLED))
+                        g_warning ("Error retrieving app filter for user %s: %s",
+                                   act_user_get_user_name (user),
+                                   error->message);
+
                 return FALSE;
         }
```